packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -531,6 +552,30 @@ describe('Combobox', () => {
         await disconnect();
     });
 
+    it('when typing in value with inline autocomplete, option at bottom of list scrolls into view', async () => {
+        const { element, connect, disconnect } = await setupWithManyOptions(
+            undefined,
+            undefined,
+            ComboboxAutocomplete.inline
+        );
+        await connect();
+        await waitForUpdatesAsync();
+
+        const lastOption = element.options[element.options.length - 1]!;
+        await clickAndWaitForOpen(element);
+        updateComboboxWithText(element, '1');
+        await waitForUpdatesAsync();
+        // This second call seems necessary to allow the requestAnimationFrame in the FAST Combobox implementation
+        // time to run for the first update.
+        await waitForUpdatesAsync();
```

Comment:
At this point (before updating the text to `1000`) should we asset that the option is not visible?

Reply:
Hmmm...not sure how I feel about that...that's basically just an assert to make sure the setup code is proper, as opposed to something related to the issue itself. If you would still like to have the assert there I'm not that against it, I just don't feel strongly about adding it.

Reply:
I'd like the assertion just because I'm less confident about tests initial conditions because we do not control the viewport. It's possible for some reason that the browser window is very long and the line is already in the viewport. We don't control that initial condition / test set-up isn't controlling for that. So a clear assertion before and after about the viewport state makes me more confident this is testing a specific change in behavior.

Reply:
> that's basically just an assert to make sure the setup code is proper,

yep! exactly lol

Reply:
Done.